### DISCUSSION TITLE
[Tutorial] CSS Styling Files Not Found As Stated in the Tutorial

### DISCUSSION THREAD

**Discussion by songyang-dev** [vercel/next.js#43590] [Help]
On this step of the basic [Next.js tutorial](https://nextjs.org/learn/basics/assets-metadata-css/css-styling), the tutorial suggests that the stylesheets can be separately found in a folder called `styles`. But there is no such file in the project. The styles are all included inside `index.js`. I suspect this tutorial has not been updated to the most recent Next version.

```
[23:16:57] song@DESKTOP-CBDKVRK:~/projects/web/nextjs-blog$ npm list
nextjs-blog@ /home/song/projects/web/nextjs-blog
├── next@13.0.4
├── react-dom@18.2.0
└── react@18.2.0
```

**Comment by GabenGar** [comment]
But tutorial specifically says to create it:
>You can place the global CSS file anywhere and use any name. So let’s do the following:
>Create a top-level `styles` directory and a `global.css` file.
>Add the following CSS inside `styles/global.css`. This code resets some styles and changes the color of the a tag:

**Reply by songyang-dev** [reply]
This is written in the introduction of the styles section, before the passage you quoted. It assumes the `styles` folder is already there.
> CSS Styling
> Let’s now talk about CSS styling.
> 
> As you can see, our index page (http://localhost:3000/) already has some styles. If you look at your file structure, you'll see a folder called styles with two CSS files: a global stylesheet (globals.css), and a CSS module (Home.module.css).
